testing/src/main/java/io/grpc/testing/TestUtils.java

```diff
@@ -217,11 +217,18 @@ public static File loadCert(String name) throws IOException {
    * Creates an SSLSocketFactory which contains {@code certChainFile} as its only root certificate.
    */
   public static SSLSocketFactory newSslSocketFactoryForCa(File certChainFile) throws Exception {
+    return newSslSocketFactoryForCa(new FileInputStream(certChainFile));
```

Comment:
I was hoping that I could just pretend it wasn't a problem.  I miss my try-with clause